.github/workflows/ci.yml

```diff
@@ -55,8 +55,10 @@ jobs:
   automerge:
     needs: test
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
     steps:
-      - uses: fastify/github-action-merge-dependabot@v2.7.1
-        if: ${{ github.actor == 'dependabot[bot]' && github.event_name == 'pull_request' }}
+      - uses: fastify/github-action-merge-dependabot@v3
         with:
-          github-token: ${{secrets.github_token}}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
          target: minor
```

Reply:
I'm personally not a fan of using minor as default. Most of these packages have full test coverage. A dependency with a  semver major change which passes the tests is for most practical purposes not a breaking change for the consumer. The downside of setting minor as default is the additional manual work required.